wagtail/tests/settings.py

```diff
@@ -91,12 +91,12 @@
     'wagtail.tests.snippets',
     'wagtail.tests.routablepage',
     'wagtail.tests.search',
-    'wagtail.contrib.wagtailstyleguide',
-    'wagtail.contrib.wagtailsitemaps',
-    'wagtail.contrib.wagtailroutablepage',
-    'wagtail.contrib.wagtailfrontendcache',
-    'wagtail.contrib.wagtailapi',
-    'wagtail.contrib.wagtailsearchpromotions',
+    'wagtail.contrib.styleguide',
+    'wagtail.contrib.sitemaps',
+    'wagtail.contrib.routablepage',
+    'wagtail.contrib.frontendcache',
+    'wagtail.contrib.api',
+    'wagtail.contrib.searchpromotions',
```

Comment:
First of all, there is already an underscore in Wagtail’s `table_block` contrib module. So either way, we have to do something to make naming more consistent.

I think the confusion has to do with Python’s history. At first, there was no rule for naming, so we ended up with module names like HTMLParser or ConfigParser and methods like assertEqual. Then rules were added in 2001 with PEP8, and a lot of Python modules & methods were renamed, but it is still messy. For example, ConfigParser was renamed to configparser or assertEqual etc. weren’t renamed, probably because they were copied from Java and the habit stayed (the backwards compatibility excuse written a PEP8 is obsolete, now that we know how to make deprecations).

PEP8 itself isn’t very consistent, since [it discourages underscores in module names but allow them for readability](https://www.python.org/dev/peps/pep-0008/#package-and-module-names), while [insisting on adding underscores to improve readability for functions, method and constant names](https://www.python.org/dev/peps/pep-0008/#function-names).
This is misleading for everyone and it’s pointless IMO to want to save one or two characters by decreasing readability and consistency.